src/fft.rg

```diff
@@ -144,133 +152,435 @@ function fft.generate_fft_interface(itype, dtype)
     return iface.get_tunable(DEFAULT_TUNABLE_NODE_COUNT)
   end
 
+  -- Get the number of GPUs
   __demand(__inline)
   task iface.get_num_local_gpus()
     return iface.get_tunable(DEFAULT_TUNABLE_LOCAL_GPUS)
   end
 
-  __demand(__inline)
+  --Task to return pointer to plan. Takes plan region and returns pointer to plan. Calls get_base_plan made above.
+   __demand(__inline)
   task iface.get_plan(plan : region(ispace(int1d), iface.plan), check : bool) : &iface.plan
   where reads(plan) do
-    -- Hack: 3Bneed to use raw access to circument CUDA checker here.
-    var pr = __physical(plan)[0]
+    format.println("In get_plan...")
```

Comment:
This section of code has a bunch of comments that seem to be summarizing basic language features. I would remove all comments that either explain how Regent features work (if this is not obvious, the place to update is the Regent language reference), and also remove all comments that summarize or restate code without adding detail. I'm on the fence about "section" comments that mark out large sections of the code; they can be helpful but I don't think this particular function is complicated enough to warrant it?

After you do that, please double check the diff and avoid reformatting things that don't need to be reformatted. Ideally there should be a diff IFF the code actually changed.

Reply:
To cover in my comment scrub